WordPress/src/main/java/org/wordpress/android/models/Capability.java

```diff
@@ -0,0 +1,22 @@
+package org.wordpress.android.models;
+
+/**
+ * Used to decide what can the current user do in a particular blog
+ * A list of capabilities can be found in: https://codex.wordpress.org/Roles_and_Capabilities#Capabilities
+ */
+public enum Capability {
+    EDIT_USERS("edit_users"), // Check if user can change another user's role
+    LIST_USERS("list_users"), // Check if user can visit People page
+    PROMOTE_USERS("promote_users"); // Check if user can invite another user
+
+    private final String label;
+
+    Capability(String label) {
+        this.label = label;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
The answer above the one you've shared (http://stackoverflow.com/a/13291109) with the more up votes kind of contradicts your point. Having said that, I'll make the change, just to make sure we avoid the confusion, it's not a big deal.